docs/content/server/advanced_setup.md

```diff
@@ -39,3 +39,31 @@ my_secret_value
 ```
 
 This is useful when running on docker so you can copy the secrets into the container without exposing them in the Dockerfile.
+
+## Database selection
+
+### SQLite
+
+By default, the server uses SQLite as the database. SQLite is a file-based database that is easy to set up and use. However, it is not recommended for production use.
```

Comment:
> However, it is not recommended for production use.

Should we remove this? It's definitely okay to use it on production unless you are hosting a SaaS based on this on the cloud, where hundred of users might be using it at the same time ? SQLite is really good on read, it's just crappy on concurrent writes. With the current state of the system I doubt the database will be the bottleneck.

Reply:
The purpose was to point out auth only works on Postgres but that isn't super clear here. Fixed